From b823d2d9a9e870e035e66d00d2faeadec35da35c Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Fri, 12 Jul 2024 09:42:00 +0200
Subject: [PATCH 1162/1695] drivers: clk: stm32mp2: fix header file inclusion
 order

Fix order of header files inclusions in stm32mp21 and stm32mp25 clock
drivers.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 9465480669656e2c97146902cafd279dfce91300 drivers: clk: stm32mp2: fix header file inclusion order]
Change-Id: I1e01752b70cbe015d2372933395f1b503ef42de5
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/393547
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Reviewed-by: Gabriel FERNANDEZ <gabriel.fernandez@foss.st.com>
---
 core/drivers/clk/clk-stm32mp21.c | 8 ++++----
 core/drivers/clk/clk-stm32mp25.c | 8 ++++----
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/core/drivers/clk/clk-stm32mp21.c b/core/drivers/clk/clk-stm32mp21.c
index b025aafec..c5dde973d 100644
--- a/core/drivers/clk/clk-stm32mp21.c
+++ b/core/drivers/clk/clk-stm32mp21.c
@@ -5,7 +5,6 @@
 
 #include <assert.h>
 #include <config.h>
-#include <libfdt.h>
 #include <drivers/clk.h>
 #include <drivers/clk_dt.h>
 #include <drivers/stm32_bsec.h>
@@ -13,12 +12,13 @@
 #include <drivers/stm32mp21_rcc.h>
 #include <dt-bindings/clock/st,stm32mp21-rcc.h>
 #include <dt-bindings/clock/stm32mp21-clksrc.h>
-#include <kernel/boot.h>
-#include <kernel/delay.h>
-#include <kernel/pm.h>
 #include <initcall.h>
 #include <io.h>
+#include <kernel/boot.h>
+#include <kernel/delay.h>
 #include <kernel/dt.h>
+#include <kernel/pm.h>
+#include <libfdt.h>
 #include <mm/core_memprot.h>
 #include <stdio.h>
 #include <stm32_sysconf.h>
diff --git a/core/drivers/clk/clk-stm32mp25.c b/core/drivers/clk/clk-stm32mp25.c
index 8078d1fb3..4d6937875 100644
--- a/core/drivers/clk/clk-stm32mp25.c
+++ b/core/drivers/clk/clk-stm32mp25.c
@@ -5,7 +5,6 @@
 
 #include <assert.h>
 #include <config.h>
-#include <libfdt.h>
 #include <drivers/clk.h>
 #include <drivers/clk_dt.h>
 #include <drivers/stm32_bsec.h>
@@ -13,12 +12,13 @@
 #include <drivers/stm32mp25_rcc.h>
 #include <dt-bindings/clock/stm32mp25-clks.h>
 #include <dt-bindings/clock/stm32mp25-clksrc.h>
-#include <kernel/boot.h>
-#include <kernel/delay.h>
-#include <kernel/pm.h>
 #include <initcall.h>
 #include <io.h>
+#include <kernel/boot.h>
+#include <kernel/delay.h>
 #include <kernel/dt.h>
+#include <kernel/pm.h>
+#include <libfdt.h>
 #include <mm/core_memprot.h>
 #include <stdio.h>
 #include <stm32_sysconf.h>
-- 
2.47.2

